key-manager/src/main/scala/org/bitcoins/keymanager/config/KeyManagerAppConfig.scala

```diff
@@ -0,0 +1,60 @@
+package org.bitcoins.keymanager.config
+
+import java.nio.file.Path
+
+import com.typesafe.config.Config
+import org.bitcoins.core.config.NetworkParameters
+import org.bitcoins.core.util.FutureUtil
+import org.bitcoins.crypto.AesPassword
+import org.bitcoins.db._
+import org.bitcoins.keymanager.WalletStorage
+
+import scala.concurrent.{ExecutionContext, Future}
+
+case class KeyManagerAppConfig(
+    private val directory: Path,
+    private val confs: Config*)(implicit val ec: ExecutionContext)
+    extends AppConfig {
+
+  override def configOverrides: List[Config] = confs.toList
+
+  override type ConfigType = KeyManagerAppConfig
+
+  override def newConfigOfType(
+      configOverrides: Seq[Config]): KeyManagerAppConfig =
+    KeyManagerAppConfig(directory, configOverrides: _*)
+
+  override def moduleName: String = "keymanager"
+
+  override def baseDatadir: Path = directory
+
+  lazy val networkParameters: NetworkParameters = chain.network
+
+  /** The path to our encrypted mnemonic seed */
+  lazy val seedPath: Path = {
+    baseDatadir.resolve(WalletStorage.ENCRYPTED_SEED_FILE_NAME)
+  }
+
+  override def start(): Future[Unit] = FutureUtil.unit
+
+  lazy val aesPasswordOpt: Option[AesPassword] = {
+    val passOpt = config.getStringOrNone("bitcoin-s.key-manager.aesPassword")
```

Comment:
Shouldn't these be incorrect now? Also a little concerning that unit tests are failing from this...

Reply:
The `moduleName` is only used for db stuff so it doesn't effect anything for the key manager app config

Reply:
ok but why not be consistent? now I need to remember configuration is `key-manager` and database connections need to be done with `keymanager`

Reply:
We even use `moduleName` for database configuration it seems. While there isn't currently a database component to the key manager, there may be times in the future where we would want one 

https://github.com/bitcoin-s/bitcoin-s/blob/2de17bb4e47961fd3fe4e55d75909b42470e3c59/db-commons/src/main/scala/org/bitcoins/db/AppConfig.scala#L165-L164